src/sentry/search/events/datasets/discover.py

```diff
@@ -677,6 +677,109 @@ def function_converter(self) -> Mapping[str, SnQLFunction]:
+                                                                                                            "spans_group"
+                                                                                                        ],
+                                                                                                    ],
+                                                                                                ),
+                                                                                            ],
+                                                                                        ),
+                                                                                    ),
+                                                                                    Column(
+                                                                                        "spans.exclusive_time"
+                                                                                    ),
+                                                                                    Column(
+                                                                                        "spans.op"
+                                                                                    ),
+                                                                                    Column(
+                                                                                        "spans.group"
+                                                                                    ),
+                                                                                ],
+                                                                            )
+                                                                        ],
+                                                                    ),
+                                                                    args["multiplier"],
+                                                                ],
+                                                            ),
+                                                            args["start_offset"],
+                                                        ],
+                                                    ),
+                                                    args["bucket_size"],
+                                                ],
+                                            ),
+                                        ],
+                                    ),
+                                    args["bucket_size"],
+                                ],
+                            ),
+                            args["start_offset"],
+                        ],
+                        alias,
+                    ),
+                    default_result_type="number",
+                    private=False,
```

Comment:
Let's make sure we mark this as private so it's not exposed to discover unexpectedly

Reply:
I made it private 👍 Out of curiosity, if the endpoint is behind a feature flag, why do we need to make it private if technically anyone without the feature flag wouldn't be able to call it?

Reply:
This list of functions is exposed via the API. Making it private means that only the endpoints that we've explicitly stated can use it will get access. If you leave it public, the histogram function will be accessible via the `eventsv2` endpoint that discover uses. This is important because some of these functions have larger impact on the overall system, like this histogram function, and we want to limit access to it to only places that make sense.

Reply:
am I missing anything here, after making it private I'm not able to call it and the tests are failing

Reply:
you'll want to add your function to the `acl` defined in the `HistogramQueryBuilder` [here](https://github.com/getsentry/sentry/blob/229100302e0dad188a331a17095030c1e706f4c3/src/sentry/search/events/builder.py#L1394)

Reply:
oh, I seeeeee. kk, done :) ty!